tests/framework-tests/core/html/FluidPromises-test.html

```diff
@@ -17,10 +17,18 @@
         <script type="text/javascript" src="../../../../src/framework/core/js/ModelTransformation.js"></script>
         <script type="text/javascript" src="../../../../src/framework/core/js/ModelTransformationTransforms.js"></script>
 
+
+        <!-- This is required not for the tests themselves, but for the coverage client -->
+        <script type="text/javascript" src="../../../../src/framework/core/js/FluidDocument.js"></script>
```

Comment:
Is this js file still required for the coverage client? This comment also applies to other files that have this file added for the need of the coverage client.